.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  push:
+    branches:
+    - release
+
+jobs:
+  Release:
+    name: Release
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set Node.js 10.13.0
+      uses: actions/setup-node@master
+      with:
+        node-version: 10.13.0
+
+    - name: Install dependencies
+      run: yarn
+
+    - name: Version
+      run: yarn lerna:version
+
+    - name: Create pull request
+      uses: gha-utilities/init-pull-request@v0.0.6
+      with:
+        pull_request_token: ${{ secrets.GIT_TOKEN }}
+        head: release
+        base: master
+        title: '[Release] Release merge request'
```

Comment:
Can we follow the same commit syntax, say `chore: Release a new version`.
Icing on the cake would be if we could get the version and add it to title or description, like `chore: v.0.5.0 release`